### PR TITLE
feat: add `math/base/special/minabsf`

### PR DESCRIPTION
**Pull request by gunjjoshi**
## Description

> What is the purpose of this pull request?

This pull request:

-   adds `math/base/special/minabsf`, which would be the single-precision variant for [`math/base/special/minabs`](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/math/base/special/minabs).

```c
float stdlib_base_minabsf( const float x, const float y );
```

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   resolves a part of #649.

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

File: lib/node_modules/@stdlib/math/base/special/minabsf/docs/repl.txt [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    floating-point number.
+
+    If any argument is `NaN`, the function returns `NaN`.
+
+    When an empty set is considered a subset of the extended reals (all real
+    numbers, including positive and negative infinity), positive infinity is the
+    greatest upper bound. Similar to zero being the identity element for the sum
+    of an empty set and to one being the identity element for the product of an
+    empty set, positive infinity is the identity element for the minimum, and
+    thus, if not provided any arguments, the function returns positive infinity.
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/special/minabsf/docs/repl.txt]: The parameters are mandatory per the types and I am getting `NaN` back when calling it without any arguments. 

```suggestion
```

The note makes sense for e.g. `math/base/special/maxabsn` and `math/base/special/minabsn`, which have a variadic number of parameters, but should be removed here.

File: lib/node_modules/@stdlib/math/base/special/minabsf/docs/types/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+// TypeScript Version: 4.1
+
+/**
+* Returns the minimum absolute single-precision floating-point number.
+*
+* ## Notes
+*
+* -   When an empty set is considered a subset of the extended reals (all real numbers, including positive and negative infinity), positive infinity is the greatest upper bound. Similar to zero being the identity element for the sum of an empty set and to one being the identity element for the product of an empty set, positive infinity is the identity element for the minimum, and thus, if not provided any arguments, the function returns positive infinity.
+*
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/special/minabsf/docs/types/index.d.ts]:
```suggestion
*
```

**Review by Planeshifter — APPROVED** [review]
Looks all good to me aside from an incorrect note, which I removed. Will merge shortly after CI has passed.

File: lib/node_modules/@stdlib/math/base/special/minabsf/docs/repl.txt (hunk 1 above)
Comment (gunjjoshi) [reply lib/node_modules/@stdlib/math/base/special/minabsf/docs/repl.txt]: Thanks for explaining!

**Pull request merged**